<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CANBridgeWindow</class>
 <widget class="QDialog" name="CANBridgeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAN Bridger</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Side 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSide1"/>
     </item>
     <item>
      <widget class="QCheckBox" name="ckEnableSide1">
       <property name="text">
        <string>Enable forwarding</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IDs to forward to side 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listSide1"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Side 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSide2"/>
     </item>
     <item>
      <widget class="QCheckBox" name="ckEnableSide2">
       <property name="text">
        <string>Enable forwarding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>IDs to forward to side 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listSide2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
